docs/DML-Operations-on-Carbon.md

```diff
@@ -1,197 +1,308 @@
+                                 'MULTILINE'='true', 'ESCAPECHAR'='\', 
+                                 'COMPLEX_DELIMITER_LEVEL_1'='$', 
+                                 'COMPLEX_DELIMITER_LEVEL_2'=':',
+                                 'ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary',
+                                 'DATEFORMAT'='projectjoindate:yyyy-MM-dd'
+                                 )
+  ```
+
+***
+
+# SHOW SEGMENTS
+This command is to show the segments of carbon table to the user.
+
+  ```ruby
+  SHOW SEGMENTS FOR TABLE [db_name.]table_name LIMIT number_of_segments;
+  ```
+
+### Parameter Description
+
+| Parameter | Description | Optional |
+| ------------- | -----| --------- |
+| db_name | Database name, if it is not specified then it uses current database. | YES |
+| table_name | The name of the table in provided database.| NO |
+| number_of_segments | limit the output to this number. | YES |
+
+**Example:**
+
+  ```ruby
+  SHOW SEGMENTS FOR TABLE CarbonDatabase.CarbonTable LIMIT 2;
+  ```
+
+***
+
+# DELETE SEGMENT BY ID
+
+This command is to delete segment by using the segment ID.
+
+  ```ruby
+  DELETE SEGMENT segment_id1,segment_id2 FROM TABLE [db_name.]table_name;
+  ```
+
+### Parameter Description
+
+| Parameter | Description | Optional |
+| ------------- | -----| --------- |
+| segment_id | Segment Id of the load. | NO |
+| db_name | Database name, if it is not specified then it uses current database. | YES |
+| table_name | The name of the table in provided database.| NO |
+
+**Example:**
+
+  ```ruby
+  DELETE SEGMENT 0 FROM TABLE CarbonDatabase.CarbonTable;
+  DELETE SEGMENT 0.1,5,8 FROM TABLE CarbonDatabase.CarbonTable;
+  ```
+  Note: Here 0.1 is compacted segment sequence id.  
+
+***
+
+# DELETE SEGMENT BY DATE
+This command will allow to deletes the Carbon segment(s) from the store based on the date provided by the user in the DML command. The segment created before the particular date will be removed from the specific stores.
+
+  ```ruby
+  DELETE SEGMENTS FROM TABLE [db_name.]table_name WHERE STARTTIME BEFORE [DATE_VALUE];
+  ```
+
+### Parameter Description
+
+| Parameter | Description | Optional |
+| ------------- | -----| ------ |
+| DATE_VALUE | Valid segement load start time value. All the segments before this specified date will be deleted. | NO |
+| db_name | Database name, if it is not specified then it uses current database. | YES |
+| table_name | The name of the table in provided database.| NO |
+
+**Example:**
+
+  ```ruby
+  DELETE SEGMENTS FROM TABLE CarbonDatabase.CarbonTable WHERE STARTTIME BEFORE '2017-06-01 12:05:06';  
+  ```
+
+***
+
+# UPDATE CARBON TABLE
+This command updates the carbon table based on the column expression and optional filter conditions.
+ 
+The client node where the UPDATE command is executed should be part of the cluster.
+### Syntax
+Syntax1:
+  ```ruby
+  UPDATE <CARBON TABLE>

```

Comment:
mention `table_name` instead of `<CARBON TABLE>`. table_name is not an optional field.